grants_tagger/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.3"
```

Comment:
0.1.x above and 0.1.3 here?

Reply:
yes by x I mean any number should I use * or something else?